lib/github.server-action.ts

```diff
@@ -0,0 +1,55 @@
+import { IGitHubEvent } from "@/lib/gh_events";
+import { Octokit } from "octokit";
+
+const GITHUB_PAT: string = process.env.GITHUB_PAT || "";
```

Comment:
use `getGitHubAccessToken` method instead as it'll handle the cases where there is no GITHUB_PAT configured gracefully during development